src/lib/course.ts

```diff
@@ -32,7 +44,7 @@ function getApiV1Url(): string {
   return `https://octopus.${config.getDomain()}/api/v1`
 }
 
-export async function info(courseId: string): Promise<Course> {
+export async function info(courseId: string): Promise<Module[]> {
```

Comment:
Why course info returns an array of `Modules`? it should return `Course` structure

Reply:
Mono<List<ModuleInformation>> fetchCourseInfo(AuthContext context, String courseId);
octopus returns list of Modules

Reply:
`fetchCourseInfo` should return course information including the id and name of the course with assignments